python/metatomic_torch/metatomic/torch/ase_calculator.py

```diff
@@ -43,6 +43,32 @@
     "float64": torch.float64,
 }
 
+# `Atoms` properties that can be get with `ase.Atoms.get_property`
+# See: https://gitlab.com/ase/ase/-/blob/master/ase/calculators/abc.py#L12
+MIXIN_PROPERTIES = [
+    "free_energy",
+    # "energy",
+    # "energies",
+    # "forces",
+    # "stress",
+    # "stresses",
+    "dipole",
+    "charges",
+    "magmom",
+    "magmoms",
```

Comment:
For now I would limit properties that can be passed around to the standard ouputs (which we might want to rename to "standard data layout?"): https://docs.metatensor.org/metatomic/latest/outputs/index.html